tap_s3_csv/__init__.py

```diff
@@ -82,7 +83,7 @@ def do_sync(config, catalog, state):
             current_col_count = len(stream['schema']["properties"].items())
             total_col_count += current_col_count
             json_row_col = {"name": name, "stream_id":stream_name, "row": counter_value, "col": current_col_count}
-            LOGGER.info("EXPORTS tap-s3-csv individual_file_data_props: " + str(json_row_col))
+            grouped_logs.append("individual_file_data_props: " + str(json_row_col) + "\n")
```

Comment:
not sure if export icm supports multiple streams in the catalog, if yes, may also want to include the stream name into the log?